crates/ibc/src/core/ics04_channel/error.rs

```diff
@@ -26,6 +27,10 @@ define_error! {
             [ port_error::Error ]
             | _ | { "ics05 port error" },
 
+        Ics20Transfer
```

Comment:
The [AppModule error](https://github.com/cosmos/ibc-rs/blob/da6653e0716350311d15e6acca08cad625f2d9ff/crates/ibc/src/core/ics04_channel/error.rs#L341-L347) should be used instead. By design, we don't want core IBC to know about specific applications

Reply:
removed